api/src/main/java/org/openmrs/module/Extension.java

```diff
@@ -41,7 +41,7 @@
 	 *
 	 * @see #toExtensionId(String, MEDIA_TYPE)
 	 */
-	public static final String extensionIdSeparator = "|";
+	public static final String EXTENSION_IDSEPARATOR = "|";
```

Comment:
The whole thing looks good, but isn't it better to replace `extensionIdSeparator` with `EXTENSION_ID_SEPARATOR`? Such a detail, it also seems good without it. At least I think so. ;)

Reply:
Thanks @sacull